src/Pages/Shared/RightBar/RightBar.js

```diff
@@ -0,0 +1,160 @@
+import {
+  Box,
+  Button,
+  Checkbox,
+  Divider,
+  FormControl,
+  FormControlLabel,
+  FormLabel,
+  List,
+  ListItem,
+  ListItemButton,
+  ListItemIcon,
+  ListItemText,
+  Radio,
+  RadioGroup,
+  styled,
+  Typography,
+} from "@mui/material";
+import react, { useReducer, useState } from "react";
+import { useReducerContext } from "../../../Helpers/Context";
+import "./RightBar.css";
+
+const RightBarBox = styled(Box)(({ theme }) => ({
+  display: "flex",
+  flexDirection: "column",
+  [theme.breakpoints.down("sm")]: {
+    backgroundColor: theme.palette.text.tertiary,
+    zIndex: 2,
+    position: "relative",
+    width: "80%",
+    marginLeft: "auto",
+  },
+}));
+
+export const RightBar = () => {
+  const [checked, setChecked] = useState([0]);
+  const [sortByDate, setSortByDate] = useState("");
+  const { dispatch, labels, loading, priority, timeSort } = useReducerContext();
+
+  const handleToggle = (value, category) => () => {
+    dispatch({
+      type: category === "priority" ? "PRIORITY_FILTER" : "LABEL_FILTER",
+      filterType: category,
+      filter: value,
+    });
+    const currentIndex = checked.indexOf(value);
+    const newChecked = [...checked];
+
+    if (currentIndex === -1) {
+      newChecked.push(value);
+    } else {
+      newChecked.splice(currentIndex, 1);
+    }
+    setChecked(newChecked);
+  };
+
+  const handleRadioChange = (event) => {
+    dispatch({
+      type: event.target.value.toUpperCase(),
+    });
+    setSortByDate(event.target.value);
+    console.log(event.target.value);
```

Comment:
changes: remove logs before pushing to github

Reply:
removed !